projectforge-business/src/main/resources/I18nResources.properties

```diff
@@ -2050,12 +2103,18 @@ poll.response.title=Poll Response Page
 poll.running=Running
 poll.selectUser=Select user
 poll.state=State
+poll.templateTypeNeujahrsfeier=New Year's Party
+poll.templateTypeSommerfest=Summer Party
+poll.templateTypeTeamessen=Team Dinner
 poll.title=Title
 poll.title.add=Create new Poll
 poll.title.edit=Edit Poll
 poll.title.list=Polls
 poll.userDelegation=User delegation
 poll.yourAnswers=Your answers
+PollMultiResponseQuestion=Multiple Choice Question
```

Comment:
Nicht änderbar aufgrund einer Methode in Question.kt wenn ich es poll.multiresponse.question nenne kriege ich einen error da ein "." nicht valid ist im Variablen namen.